src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  margin: auto 0;
+
+  &__list {
+    display: flex;
+    justify-content: space-between;
+    text-decoration: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin: 0;
+    text-decoration: none;
+    list-style: none;
+
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
```

Comment:
Why you reset all margins, and after give it to each element except last? Make in another way - give margin to each elements except last